charts/cadc-tap/templates/uws-db-networkpolicy.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Values.mockdb.enabled -}}
```

Comment:
I don't think you meant mockdb here.

Reply:
Oops, thought I had fixed this one, good catch thank you